ansible/roles/hcv-haproxy-configure/handlers/main.yml

```diff
@@ -1,32 +0,0 @@
----
-
-- name: Reload haproxy
```

Comment:
We still need all these in the case that we aren't using consul template, so please do not remove them

Reply:
accidental deletion, fixed